roaring/roaring.go

```diff
@@ -4180,7 +4186,8 @@ func readOfficialHeader(buf []byte) (size uint32, containerTyper func(index uint
 		return newType
 	}
 	containerTyper = cf
-	cookie := binary.LittleEndian.Uint32(buf)
+	cookie := binary.LittleEndian.Uint32(buf) & 0xFFFFFF
+	flags = buf[3]
```

Comment:
should we be reading flags from the "official" header? i thought this was basically upstream Roaring format, which we wouldn't want to interpret differently.